engine/core/src/test/java/org/pentaho/reporting/engine/classic/core/modules/output/fast/ReportStructureValidatorTest.java

```diff
@@ -103,30 +106,33 @@ public void testNoneInlineSubReportDetection() {
 
   @Test
   public void testBarcodeDetection() {
-    ElementMetaData em = new DefaultElementMetaData( "simple-barcodes", "simple-barcodes",
-     "simple-barcodes", "simple-barcodes", false, false, false,
-     false, null, new AttributeMap<>(), new HashMap<>(), BarcodeDummyType.class,
-     BarcodeDummyType.class, null, 1 );
-    ElementTypeRegistry.getInstance().registerElement( em );
-
-    Element e = new Element();
-    e.setElementType( new BarcodeDummyType() );
-
-    SubReport sr = new SubReport();
-    sr.getReportDefinition().getItemBand().addElement( e );
-
-    MasterReport report = new MasterReport();
-    report.getReportHeader().addSubReport( sr );
-
-    ReportStructureValidator v = new ReportStructureValidator();
-    assertFalse( v.isValidForFastProcessing( report ) );
+    try {
+      ElementMetaData em = new DefaultElementMetaData( SIMPLE_BARCODES_TYPE, SIMPLE_BARCODES_TYPE,
+        SIMPLE_BARCODES_TYPE, SIMPLE_BARCODES_TYPE, false, false, false,
+        false, null, new AttributeMap<>(), new HashMap<>(), BarcodeDummyType.class,
+        BarcodeDummyType.class, null, 1 );
+      ElementTypeRegistry.getInstance().registerElement( em );
+
+      Element e = new Element();
+      e.setElementType( new BarcodeDummyType() );
+
+      SubReport sr = new SubReport();
+      sr.getReportDefinition().getItemBand().addElement( e );
+
+      MasterReport report = new MasterReport();
+      report.getReportHeader().addSubReport( sr );
+
+      ReportStructureValidator v = new ReportStructureValidator();
+      assertFalse( v.isValidForFastProcessing( report ) );
+    } finally {
+      ElementTypeRegistry.getInstance().removeElement( SIMPLE_BARCODES_TYPE );
```

Comment:
This solves the problem I was having with running locally. the Instance object `BarcodeDummyType( "simple-barcodes"`) was failing `org.pentaho.reporting.engine.classic.core.metadata.AttributeMetaDataValidationTest`